### ISSUE TITLE
Agenda View isn't clickable

### ISSUE THREAD

**Issue by stephlabou** [uc-love-data-week/uc-love-data-week.github.io#139]
[Agenda view](https://uc-love-data-week.github.io/agenda) doesn't have any clickable links

**Comment by stephlabou** [comment]
Current set up ([2024 example])(https://github.com/uc-love-data-week/uc-love-data-week.github.io/blob/main/_2024_pages/agenda.html) is to have registration links included and clickable up to and through event, but not after.

To do: mirror content on modal pop ups which include links to recordings and slides after event has ended. ([2024 example](https://github.com/uc-love-data-week/uc-love-data-week.github.io/blob/main/_includes/2024_includes/workshops_info.html))

**Comment by stephlabou** [comment]
Noting that the example files linked above could use some in-file comments for documentation. There are a lot of if/endif statements that are hard to parse as written. Documentation goal would be to clarify which section maps to which portion of web output, so easier to update in future.

**Comment by kekoziar** [comment]
@stephlabou I can work on documentation.  Just assign things to me and I'll work through them when I have time.  (coding and everything related to it is always a nice break for me).  

regarding reading the file, I agree it's not easy to read. The issue with that file is adding newline characters to make the code readable renders them as new lines in the html.  I'm not sure why it does that in some files and not others.
